chaos-monkey-dependencies/pom.xml

```diff
@@ -72,6 +72,11 @@
                 <artifactId>spring-boot-starter-validation</artifactId>
                 <version>${spring-boot.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-starter-security</artifactId>
+                <version>${spring-boot.version}</version>
+            </dependency>
```

Comment:
Spring Security should be only needed in the demo app and added only there as dependency.

If thats not possible, im sorry for bringin this up, then I would preferr the initial approach 🙈

Reply:
You are correct, it is only used for the demo. Should I move it into that demo app, or is leaving it here OK?

Reply:
I'll preferr if its moved to the demo app.